packages/documentation/copy/en/handbook-v2/More on Functions.md

```diff
@@ -329,9 +329,9 @@ Functions in JavaScript often take a variable number of arguments.
 For example, the `toFixed` method of `number` takes an optional digit count:
 
 ```ts twoslash
-function f(n: number) {
+function f(n: number, x?: number) {
   console.log(n.toFixed()); // 0 arguments
-  console.log(n.toFixed(3)); // 1 argument
+  console.log(n.toFixed(x)); // 1 argument

```

Comment:
I don't think this is a good change; the optionality of the parameter to `toFixed` was already covered by either not passing an argument, or passing one, which is what this code was doing before.